pkg/cri/config/config.go

```diff
@@ -509,5 +518,12 @@ func ValidatePluginConfig(ctx context.Context, c *PluginConfig) error {
 			return fmt.Errorf("invalid image pull progress timeout: %w", err)
 		}
 	}
+
+	// Validation for drain_exec_sync_io_timeout
+	if c.DrainExecSyncIOTimeout != "" {
+		if _, err := time.ParseDuration(c.DrainExecSyncIOTimeout); err != nil {
+			return fmt.Errorf("invalid drain exec sync io timeout: %w", err)
```

Comment:
Can be a followup as the other two timeouts didn't follow this pattern, but it looks like prior there was a pattern of using the toml name of the configuration in the error message reported to the user. So this would be:

```suggestion
			return fmt.Errorf("invalid `drain_exec_sync_io_timeout`: %w", err)
```

Reply:
updated.